comfy_extras/nodes_preview_any.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+# Preview Any - original implement from
+# https://github.com/rgthree/rgthree-comfy/blob/main/py/display_any.py
+# upstream requested in https://github.com/Kosinkadink/rfcs/blob/main/rfcs/0000-corenodes.md#preview-nodes
+class PreviewAny():
+    @classmethod
+    def INPUT_TYPES(cls):
+        return {
+            "required": {"source": ("*", {})},
```

Comment:
nit: Use `node_typing.IO.ANY` to replace `"*"` here.

Reply:
done